example/src/App.tsx

```diff
@@ -54,14 +54,20 @@ class App extends React.Component {
 
     Radar.requestLocationPermissions({ background: false });
 
-    /*
+    Radar.setForegroundServiceOptions({
+      options: {
+        title: 'Foreground service title',
+        text: 'Foreground service text'
+      }
+    });
+
     Radar.startTrackingContinuous();
 
     Radar.trackOnce().then((result) => {
       alert(JSON.stringify(result));
     });
-    */
 
+    /*
```

Comment:
Are we re-enabling this later?

Reply:
Potentially. Long-term, sample apps need a total overhaul across platforms. Right now they're just an inconsistent kitchen sink of code snippets (primary benefit right now is to make sure sample apps compile in CircleCI, which is why I added `setForegroundServiceOptions()` here)